runtime/vm/createramclass.cpp

```diff
@@ -3133,6 +3159,9 @@ internalCreateRAMClassFromROMClass(J9VMThread *vmThread, J9ClassLoader *classLoa
 		if (J9_ARE_ALL_BITS_SET(valueTypeFlags, J9ClassContainsUnflattenedFlattenables)) {
 			classFlags |= J9ClassContainsUnflattenedFlattenables;
 		}
+		if (J9_ARE_ALL_BITS_SET(valueTypeFlags, J9ClassCanSupportFastSubstitutability)) {
```

Comment:
this should be placed within the `if (J9_ARE_ALL_BITS_SET(romClass->modifiers, J9AccValueType)) {` block above

Reply:
Should I move the `if (J9_ARE_ALL_BITS_SET(valueTypeFlags, J9ClassContainsUnflattenedFlattenables)) {` one within that scope as well, that flag should only apply to value types right?

Reply:
`J9ClassContainsUnflattenedFlattenables` can apply to non valuetypes

Reply:
I've made the change